Documentation/tutorials/EdgeBridgeTutorialAppFinal/app/build.gradle

```diff
@@ -44,25 +45,21 @@ spotless {
 }
 
 dependencies {
-
     implementation 'androidx.appcompat:appcompat:1.5.1'
     implementation 'com.google.android.material:material:1.6.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
     implementation 'androidx.navigation:navigation-fragment:2.5.2'
     implementation 'androidx.navigation:navigation-ui:2.5.2'
-    testImplementation 'junit:junit:4.13.2'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
 
     // Add dependencies on Adobe Experience Platform Mobile extensions
-    implementation 'com.adobe.marketing.mobile:assurance:2.0.0'
-    implementation 'com.adobe.marketing.mobile:core:2.0.0'
-    implementation 'com.adobe.marketing.mobile:lifecycle:2.0.0'
-    implementation 'com.adobe.marketing.mobile:edge:2.0.0'
-    implementation 'com.adobe.marketing.mobile:edgeconsent:2.0.0'
-    implementation 'com.adobe.marketing.mobile:edgeidentity:2.0.0'
+    implementation 'com.adobe.marketing.mobile:assurance:3.0.0'
```

Comment:
Do we want to promote using bom?

Reply:
For the tutorial app the dependencies don't change as often as for the test app, using bom here sounds good to me.